rules/kubecf/def.bzl

```diff
@@ -0,0 +1,70 @@
+"""
```

Comment:
Needs comments for everything

Reply:
Fixed with https://github.com/cloudfoundry-incubator/kubecf/pull/571/commits/73796132ab74bf44ac495e988e2cc4ce5f8d5e14.